types/nightwatch/index.d.ts

```diff
@@ -594,6 +594,12 @@ export interface NightwatchOptions {
      */
     skiptags?: string;
 
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/nightwatch/index.d.ts;
  Line: 597;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
@Ykumar1415 Can you replace tabs with spaces here?

Reply:
sir is it fine now ?